scss/custom/_modal.scss

```diff
@@ -25,19 +22,14 @@
   .modal-header,
   .modal-footer {
     // Fixed for older phones
```

Comment:
The comment is misleading then. The "Fix for older phones" will be added by the prefixer (hopefully)